internal/cli/run/run.go

```diff
@@ -50,20 +48,20 @@ func init() {
 	cmd := root.Command("run", "Run a test group or OONI Run link")
 
 	var nettestGroupNamesBlue []string
+	var ctx *ooni.Context
+	var network *database.Network
+
 	for name := range groups.NettestGroups {
 		nettestGroupNamesBlue = append(nettestGroupNamesBlue, color.BlueString(name))
 	}
 
-	nettestGroup := cmd.Arg("name",
-		fmt.Sprintf("the nettest group to run. Supported tests are: %s, or nothing to run them all",
-			strings.Join(nettestGroupNamesBlue, ", "))).String()
-
 	noCollector := cmd.Flag("no-collector", "Disable uploading measurements to a collector").Bool()
 	collectorURL := cmd.Flag("collector-url", "Specify the address of a custom collector").String()
 	bouncerURL := cmd.Flag("bouncer-url", "Specify the address of a custom bouncer").String()
 
-	cmd.Action(func(_ *kingpin.ParseContext) error {
-		ctx, err := root.Init()
+	cmd.Action(func(kp *kingpin.ParseContext) error {
```

Comment:
It seems `kp` is still unused?

Reply:
That is true, I suppose it can be underscored.